Symfony/CS/Fixer/Contrib/PhpUnitDedicateAssertFixer.php

```diff
@@ -0,0 +1,221 @@
+     *
+     * @return int|int[] indexes of assert call, test call and positive flag, or last index checked.
+     */
+    private function getAssertCandidate(Tokens $tokens, $assertCallIndex)
+    {
+        $content = strtolower($tokens[$assertCallIndex]->getContent());
+        if ('asserttrue' === $content) {
+            $isPositive = 1;
+        } elseif ('assertfalse' === $content) {
+            $isPositive = 0;
+        } else {
+            return $assertCallIndex;
+        }
+
+        // test candidate for simple calls like: ([\]+'some fixable call'(...))
+        $assertCallOpen = $tokens->getNextMeaningfulToken($assertCallIndex);
+        if (!$tokens[$assertCallOpen]->equals('(')) {
+            return $assertCallIndex;
+        }
+
+        $testNsToken = false;
+        $test = $tokens->getNextMeaningfulToken($assertCallOpen);
+
+        if (!$tokens[$test]->isGivenKind(array(T_EMPTY, T_STRING))) {
+            if (!$tokens[$test]->isGivenKind(T_NS_SEPARATOR)) {
+                return $test;
+            }
+
+            $testNsToken = $test;
+            $test = $tokens->getNextMeaningfulToken($test);
+            if (!$tokens[$test]->isGivenKind(T_STRING)) {
+                return $test;
+            }
+        }
+
+        $testOpen = $tokens->getNextMeaningfulToken($test);
+        if (!$tokens[$testOpen]->equals('(')) {
+            return $testOpen;
+        }
+
+        $testClose = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $testOpen);
+
+        $assertCallClose = $tokens->getNextMeaningfulToken($testClose);
+        if (!$tokens[$assertCallClose]->equalsAny(array(')', ','))) {
+            return $assertCallClose;
+        }
+
+        return array(
+            $isPositive,
+            $assertCallIndex,
+            $assertCallOpen,
+            $testNsToken,
+            $test,
+            $testOpen,
+            $testClose,
+            $assertCallClose,
+        );
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param array  $assertIndexes
+     *
+     * @return int index up till processed, number of tokens added
+     */
+    private function fixAssert(Tokens $tokens, array $assertIndexes)
+    {
+        list(
+            $isPositive,
+            $assertCallIndex,
+            $assertCallOpen,
+            $testNsToken,
+            $test,
+            $testOpen,
+            $testClose,
+            $assertCallClose
+        ) = $assertIndexes;
+
+        $content = strtolower($tokens[$test]->getContent());
+        if (array_key_exists($content, $this->fixMap)) {
+            if (false !== $this->fixMap[$content][$isPositive]) {
+                $tokens[$assertCallIndex]->setContent($this->fixMap[$content][$isPositive]);
+                $this->removeFunctionCall($tokens, $testNsToken, $test, $testOpen, $testClose);
+            }
+
+            return $assertCallClose;
+        }
+
+        switch ($content) {
```

Comment:
>yes

not sure I follow, which one would you prefer, `multi line if` or `in_array`? 

>also, what if content doesn't match that cases?

`content` is made lower case on line 160 so this should be no issue :)